tests/unit_tests/test_wave_02.py

```diff
@@ -34,7 +34,11 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("electronics")
 
-    raise Exception("Complete this test according to comments below.")
+    #raise Exception("Complete this test according to comments below.")
+    assert item_a != items
```

Comment:
Close! `!=` is testing for non-equality. We'd want to check that the `item_a`, `item_b`, and `item_c `are `not in items` since none of their categories are "electronics".